test/unittests/execute_call_test.cpp

```diff
@@ -205,6 +233,36 @@ TEST(execute_call, call_indirect_uninited_table)
     EXPECT_THAT(execute(module, 3, {4}), Traps());
 }
 
+TEST(execute_call, call_indirect_shared_stack_space)
```

Comment:
Seems to be a dup of `imported_functions_call_indirect` ?

Reply:
Both are simplified variants of it. They can detect an implementation defect in #572, so I keep them as they are.